docs/setup-and-installing.md

```diff
@@ -83,20 +83,20 @@ If you are using Rider, install the [**Unity Support** plugin](https://github.co
 
 <%(/Expandable)%>
 
-### Validate your installation
+<%(#Expandable title="Having problems?")%>
```

Comment:
Would be nice if we could offer some pointers on how to debug their own problems

Reply:
I can add a link to the troubleshooting page?

Reply:
If its potentially useful for setup problems

Reply:
not really (at least right now)

but don't know what else to add